presetSymbols.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Should we keep this file in the repo?

Reply:
We need to keep these in for now as removing them means changing the external API, and feeds will break on adapter updates.